### PR TITLE
[PR-2221] Replace deprecated set-output command

### PR DESCRIPTION
**Pull request by cbookg**

### CODE REVIEW

File: .github/actions/bump-skip/action.yml [hunk 1]
```diff
@@ -24,5 +24,5 @@ runs:
           IS_BUMP=yes
         fi
         echo "IS_BUMP=$IS_BUMP"
-        echo ::set-output name=is-bump::$IS_BUMP
+        echo is-bump=$IS_BUMP >> $GITHUB_OUTPUT
```
Comment (zloery) [inline .github/actions/bump-skip/action.yml:27]: Does this need to be quoted to substitute the value? I forget the exact rules for GHA syntax

**Review by zloery — APPROVED** [review]

**Review by ddietterich — APPROVED** [review]
Thank you!!

File: .github/actions/bump-skip/action.yml (hunk 1 above)
Comment (cbookg) [reply .github/actions/bump-skip/action.yml:27]: Not sure, so I kept as-is.  But nothing has failed...

Comment (cbookg) [reply .github/actions/bump-skip/action.yml:27]: a bunch of passing tests made use of non-quoted values.

**Pull request merged**